extensions/amp-video/validator-amp-video.protoascii

```diff
@@ -185,9 +185,8 @@ tags: {  # <amp-video> in amp-story
   }
   attrs: {
     name: "volume"
-    # Disallow 0 to prevent shipping unnecessary audio track when the video is muted.
-    # Volume must be in the range [0.1, 1].
-    value_regex: "^((0?\\.[1-9]+)?|1(\\.0*)?)$"
+    # Volume between 0 and 1 (inclusive).
+    value_regex: "0(\\.[0-9]+)?|1(\\.0+)?"
```

Comment:
This is being done in https://github.com/ampproject/amphtml/pull/37376, I think you should be able to remove it from your PR :))

Reply:
oh cool, thanks!